Source/WTF/wtf/WeakPtr.h

```diff
@@ -76,10 +80,11 @@ template<typename Counter = EmptyCounter> class WeakPtrImpl : public ThreadSafeR
         , m_wasConstructedOnMainThread(isMainThread())
 #endif
     {
-        Counter::increment();
+        m_counter.increment();
     }
 
     void* m_ptr;
+    NO_UNIQUE_ADDRESS Counter m_counter;
```

Comment:
Rather than including Counter as a member, we can specialize WeakPtrImpl and store data there.

Reply:
OK, apply this change.